conf/evolutions/default/8.sql

```diff
@@ -0,0 +1,60 @@
+#
```

Comment:
Currently, there are files only till 5.sql in the master branch. Rename it to 6.sql